README.md

```diff
@@ -1,6 +1,33 @@
-# curriculum-cr-training-mobile-menu
+# Portfolio for review
 
+> This is a Desktop and Mobile version of my Portfolio Website.
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+## Below is a screenshot showing my Portfolio
+![gif](Portfolio.png)
+## Built With
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+- HTML
+- CSS
+- VSCode
+
+## Author
+
```

Comment:
- [ ] [OPTIONAL]  Some recruiters may not be as technical and might need help on how to have the app running. Please include the steps in your README